sky/server/requests/executor.py

```diff
@@ -390,6 +393,114 @@ def _request_execution_wrapper(request_id: str,
             logger.info(f'Request {request_id} finished')
 
 
+async def execute_request(request: api_requests.Request):
```

Comment:
Can we rename to clarify that this uses a coroutine rather than the typical request submission?
```suggestion
async def execute_request_coroutine(request: api_requests.Request):
```

Reply:
The `async` keyword implies the same semantic here as "coroutine" and there would be a lint failure if `execute_request` is not called with `await` or asyncio primitives. For request that is executed in process executor, we use `schedule_request`. So maybe it is okay to keep the current one for brevity, wdyt?

Reply:
I still would like some distinction. If not in the name, in a comment, but I think the name is better.

My concern is not that this function must be run in a coroutine. As you say, that's clear from the `async` keyword. It's more about clarifying the type of request (a "coroutine" request vs a normal executor-based request).

The main concern: If I am new to skypilot and look at this function, I might expect that _all requests_ are going to be executed using this function. In fact, only coroutine requests use this. For other requests, there is a totally different code path that the executor uses to execute the request. I may not realize that the executor does not run requests using coroutines.

Reply:
Sounds good, Thanks for the careful consideration!